internal/config/config.go

```diff
@@ -452,10 +452,14 @@ func New(fileHandler file.Handler, name string, fetcher attestationconfigapi.Fet
 		c.Provider.OpenStack.Password = openstackPassword
 	}
 
-	// Backwards compatibility: configs without the field `microserviceVersion` are valid in version 2.6.
-	// In case the field is not set in an old config we prefill it with the default value.
-	if c.MicroserviceVersion == "" {
-		c.MicroserviceVersion = Default().MicroserviceVersion
+	// Backwards compatibility: configs without the field `deployCSIDriver` are valid in version 2.8.
+	// TODO (msanft): v2.9. Remove after v2.9 release.
+	if c.Provider.AWS != nil && c.Provider.AWS.DeployCSIDriver == nil {
+		fmt.Fprintln(
+			os.Stderr,
+			"WARNING: 'provider.aws.deployCSIDriver' will be required in an upcoming version. Setting to 'false'.",
```

Comment:
```suggestion
			"WARNING: 'provider.aws.deployCSIDriver' not set. The key will be required in v2.10. Defaulting to 'false'.",
```
We have clear plans on when we want to enforce this value. Maybe add that here?